### DISCUSSION TITLE
Using pipe to write stdin causes srslog error

### DISCUSSION THREAD

**Discussion by john-dooe** [srsran/srsRAN_Project#565] [General Help]
I need to use python subprocess to run gnb, and write data to stdin. So I used `gnb = subprocess.Popen(cmd , stdin=subprocess.PIPE)`，but it causes srslog error:

```
Lower PHY in quad executor mode.

--== srsRAN gNB (commit e6c26f5) ==--

Connecting to AMF on 127.0.0.5:38412
srsLog error - The backend queue size is about to reach its maximum capacity of 8192 elements, new log entries will get discarded.
Consider increasing the queue capacity.
srsGNB ERROR: CU-CP failed to connect to AMF
```

I used pipe in bash, and it casues the same error.

```bash
echo -e "t\n" | sudo ./srsran/build/apps/gnb/gnb -c ./config/srsran/gnb_rf_b200_tdd_n78_20mhz.yml
```

How to solve this error?

**Comment by pgawlowicz** [comment]
Not sure about the usage of the pipe, but if you just want to enable the metric trace in gnb then add the following to your config:
```
metrics: 
  autostart_stdout_metrics: 1
```

**Reply by pgawlowicz** [reply]
if you want to receive metrics (formated in json) with Python then check [here ](https://github.com/srsran/srsRAN_Project/discussions/539#discussioncomment-9008517)

**Reply by pgawlowicz** [reply]
@john-dooe did it solve your issues?
